### PR TITLE
Add .css and .js file to list of available content types for freeze

### LINKED ISSUE #96: Add js and css content types to list of available content types for freeze

### PR DESCRIPTION
**Pull request by tormozz48**
Resolved #96 

@tadatuta @blond @doochik 

Please review it

### CODE REVIEW

**Comment by blond** [comment]
I'm not sure that's the right way. Maybe better to add API with which you can change this list of content types?

**Comment by tadatuta** [comment]
@blond it's possible now with env variable

File: lib/freeze.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ const contentTypes = {
     '.swf': 'application/x-shockwave-flash',
     '.ttf': 'application/x-font-ttf',
     '.woff': 'application/x-font-woff',
-    '.woff2': 'application/font-woff2'
+    '.woff2': 'application/font-woff2',
```
Comment (tadatuta) [inline lib/freeze.js]: types should be sorted in alphabetical order

**Comment by blond** [comment]
> @blond it's possible now with env variable

it's not API :(

**Comment by tadatuta** [comment]
@tormozz48 tests should pass after rebase

**Comment by arikon** [comment]
:-1: for this change. It should be configurable through the config.

**Comment by arikon** [comment]
@tormozz48 :-1: for merging without proper review

**Pull request merged**